AirCasting/SearchAndFollow/SearchLocation/SearchView.swift

```diff
@@ -0,0 +1,53 @@
+// Created by Lunar on 14/02/2022.
+//
+
+import SwiftUI
+import AirCastingStyling
+
+
+struct SearchView: View {
+    @StateObject var viewModel: SearchViewModel
+    @Binding var creatingSessionFlowContinues: Bool
+    
+    init(creatingSessionFlowContinues: Binding<Bool>) {
+        _viewModel = .init(wrappedValue: SearchViewModel())
+        self._creatingSessionFlowContinues = creatingSessionFlowContinues
+    }
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            title
+            textField
+            Spacer()
+            button
+        }.padding()
+        .sheet(isPresented: $viewModel.isLocationPopupPresented) {
+            PlacePicker(service: SearchPickerService(address: $viewModel.location))
+        }
+    }
+}
+
+// MARK: - Private View Components
+private extension SearchView {
+    var title: some View {
+        Text(Strings.SearchView.title)
+            .foregroundColor(.aircastingDarkGray)
+            .font(Font.muli(size: 24, weight: .medium))
+            .padding(.bottom, 20)
+    }
+    
+    var textField: some View {
+        createTextfield(placeholder: Strings.SearchView.placeholder,
+                        binding: $viewModel.location)
```

Comment:
I don't like that binding here. It makes viewmodel's `location` property read-write. And viewmodels should have methods as input and @Published properties as output. Maybe provide a get {} set {} binding style here and add a method to a VM for input